test/helpers.js

```diff
@@ -135,5 +153,8 @@ module.exports = {
     }, {
         email: 'john.smith2000@gmail.com',
         username: 'JohnSmith'
+    }, {
```

Comment:
Rather than add a new user object, we can re-use a single record can call toUpperCase() on the email before we do the second save.